src/rating/rating.js

```diff
@@ -22,7 +23,9 @@ angular.module('ui.bootstrap.rating', [])
 
     this.stateOn = angular.isDefined($attrs.stateOn) ? $scope.$parent.$eval($attrs.stateOn) : ratingConfig.stateOn;
     this.stateOff = angular.isDefined($attrs.stateOff) ? $scope.$parent.$eval($attrs.stateOff) : ratingConfig.stateOff;
-
+    var tmpTitles = angular.isDefined($attrs.titles)  ? $scope.$parent.$eval($attrs.titles) : ratingConfig.titles ;    
+    this.titles =   angular.isArray(tmpTitles) && tmpTitles.length > 0 ? tmpTitles  : ratingConfig.titles;
```

Comment:
Delete extra whitespace before `angular.isArray` and delete one space before `:`